daliuge-translator/dlg/dropmake/lg.graph.schema

```diff
@@ -296,58 +307,20 @@
                                 "positional": {
                                     "type": "boolean",
                                     "default": false
-                                }
-                            },
-                            "required": [
-                                "description",
-                                "name",
-                                "text",
-                                "value"
-                            ]
-                        }
-                    },
-                    "applicationArgs": {
```

Comment:
**question**: Why did you remove `applicationArgs`? I thought these were needed to pass values to the drop invocation or is my understanding wrong? I notice they're still in many of the test graphs.

Reply:
Good catch, thanks, I should not have removed applicationArgs. At some point in the future, we will drop the applicationArgs attribute, since all its elements will be moving into the "fields" attribute, but those changes have not made it to master yet.

I'll add 'applicationArgs' back until then.